server/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server.js"
```

Comment:
This is a great idea to add nodemon into the start script! I should do this too thank you